radio/src/gui/colorlcd/radio_diaganas.cpp

```diff
@@ -60,12 +60,14 @@ class RadioAnalogsDiagsWindow: public Window {
         dc->drawLine(touchState.x - 10, touchState.y + 8 - parent->top(), touchState.x + 10, touchState.y - 8- parent->top(), SOLID, 0);
       }
 
+#if !defined(SIMU)
```

Comment:
I would have left this enabled, if it compiles now, as this keeps the simu as near as possible to the transmitter UI it's simulating. I know the reason you don't want it to show is because it's not going to have any data, but the text will still be drawn where it is shown on the actual hardware. In other words, I don't like the idea making the simu behave any differently to the hardware UI/UX where it's not strictly necessary for it to function.

Reply:
Ah, I didn't realise the libui PR hadn't been merged... fair enough... still don't like this, and would prefer dummy variables declared if in simu, but at least this is compiling.

Reply:
We should mark it better once NV14 gets merged.